schema/deploy/computed_columns/form_change_as_emission_intensity_report.sql

```diff
@@ -0,0 +1,38 @@
+-- Deploy cif:computed_columns/form_change_as_emission_intensity_report to pg
+
+begin;
+
+create or replace function cif.form_change_as_emission_intensity_report(cif.form_change)
+returns cif.emission_intensity_report
+as $$
+    select
+      /**
+        Given form_data_record_id can be null for some form_change records, it is not a reliable id value for the returned project_contact record.
+        The returned id must not be null, so we use the form_change id being passed in as a parameter (multiplied by -1 to ensure we are not touching any existing records).
+        This means the id value is not going to be the correct id for the returned project_contact record, which should be ok since we're only interested
```

Comment:
```suggestion
        This means the id value is not going to be the correct id for the returned emission_intensity_report record, which should be ok since we're only interested
```